bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -111,6 +115,7 @@
 		</supported-bridge-type-refs>
 		<label>@text/dimmerLabel</label>
 		<description>@text/dimmerDescription</description>
```

Comment:
LightSource is more appropriate.

Reply:
See below..

Reply:
I think both the dimmer and the on/off thing are `ControlDevice`..